### ISSUE TITLE
Scaling MISO for Isoform prediction

### ISSUE THREAD

**Issue by ericmjonas** [ericmjonas/pywren#61]
https://miso.readthedocs.io/en/fastmiso/ is a library that does massively parallel MCMC for Isoform prediction written by a friend. He's had no end of struggles with regular users trying to make it scale on their clusters. Would be a great pywren use case.

**Comment by yarden** [comment]
Yes. I think that would be a great use case for Pywren, potentially. Each gene/event takes < 1 min, sometimes just a couple of seconds (sometimes all of the work is in I/O), but there are ~20,000 genes **x** (# of samples you sequence) events to process.

**Comment by ericmjonas** [comment]
Do you have any example data you could share?
